src/parsers/pm2ps_parser.jl

```diff
@@ -87,27 +91,49 @@ function Base.convert(::Type{BusTypes.BusType}, x::MatpowerBusTypes.MatpowerBusT
     return map[x]
 end
 
+# Disabling this because not all matpower files define areas even when bus definitions
+# contain area references.
+#function read_area!(sys::System, data; kwargs...)
+#    if !haskey(data, "areas")
+#        @info "There are no Areas in this file"
+#        return
+#    end
+#
+#    for (key, val) in data["areas"]
+#        area = Area(string(val["col_1"]))
+#        add_component!(sys, area; skip_validation = SKIP_PM_VALIDATION)
+#    end
+#end
+
```

Comment:
I want to call out that in the old code, if there were no areas defined in the file, we did not create any LoadZones.  Some matpower files don't define areas even when there are areas (and zones).  We will new create LoadZones based on whether a bus has reference to one, so should now have LoadZone objects that did not previously get created.

Reply:
This behavior is ok by me.